packages/auth/src/core/credentials/email.test.ts

```diff
@@ -228,13 +214,10 @@ describe('core/credentials/email', () => {
               clientType: RecaptchaClientType.WEB,
               version: RecaptchaVersion.ENTERPRISE
             },
-            {
-              recaptchaKey: 'mock/project/mock/site-key',
-              recaptchaConfig: { emailPasswordEnabled: true }
-            }
+            recaptchaConfigResponseEnforce
```

Comment:
Yeah, good point. Let me see how can i force the first verification to fail with MISSING_RECAPTCHA_TOKEN.

Reply:
Test added.

Reply:
Thanks for adding it, looks great! I wonder if it should be in a more generic location, like packages/auth/src/core/strategies/email_and_password.test.ts?

It would be great to add it for each flow - sign in , sign up, reset password, email link. WDYT? Would that be too repetitive? Since the fallback logic is implemented in each method (instead of a common place), i think we would benefit from individual test cases.

Reply:
Yeah, gonna add it to other flows as well, since the fallback logic is repetitive as well.